app/scripts/migrations/121.1.test.ts

```diff
@@ -0,0 +1,69 @@
+import { AccountsControllerState } from '@metamask/accounts-controller';
+import { createMockInternalAccount } from '../../../test/jest/mocks';
+import { migrate, version } from './121.1';
+
+const oldVersion = 121;
+
+const mockInternalAccount = createMockInternalAccount();
+const mockAccountsControllerState: AccountsControllerState = {
+  internalAccounts: {
+    accounts: {
+      [mockInternalAccount.id]: mockInternalAccount,
+    },
+    selectedAccount: mockInternalAccount.id,
+  },
+};
+
+describe('migration #126', () => {
```

Comment:
Missed this test description update:
```suggestion
describe('migration #121.1', () => {
```